server/lib/nl/common/constants.py

```diff
@@ -845,3 +845,155 @@
 ROOT_TOPIC = 'dc/topic/Root'
 
 PROJECTED_TEMP_TOPIC = 'dc/topic/ProjectedClimateExtremes'
+
+# Set of stat vars that need to check which facet is being used before doing
+# ranking because they have facets that are too fine in granularity.
+# This list of svs are svs in the medium embedding index that have a BLS source.
```

Comment:
Add a comment how you got this list?  Maybe link to query

Reply:
done